R/comms.R

```diff
@@ -0,0 +1,27 @@
+hipercow_speak_linux <- function(msg) {
+  invisible(system(paste0("paplay ", msg),
+                   ignore.stdout = TRUE,
+                   ignore.stderr = TRUE,
+                   wait = FALSE))
+}
+
+hipercow_speak <- function(msg, platform = Sys.info()["sysname"]) {
```

Comment:
if we don't have one already, perhaps use an `is_linux`  helper or something less gross than Sys.info()

Reply:
see windows driver's driver.R we have:

```
is_windows <- function() {
  Sys.info()[["sysname"]] == "Windows"
}
```